e2e/stack.go

```diff
@@ -36,6 +39,12 @@ type Stack struct {
 	eventListener    EventListener
 	l1BlockTime      uint64
 	prometheusCfg    *config.InstrumentationConfig
+	L1Users          []L1User
+}
+
+type L1User struct {
+	Address    common.Address
+	PrivateKey *ecdsa.PrivateKey
```

Comment:
Is this abstraction necessary? I think it would be better to add something like this when we have the need for multiple L1 users (in that case, it would probably best to keep it private or off of the `Stack` struct entirely).